show/muxcable.py

```diff
@@ -503,6 +503,42 @@ def create_json_dump_per_port_config(db, port_status_dict, per_npu_configdb, asi
     if soc_ipv4_value is not None:
         port_status_dict["MUX_CABLE"]["PORTS"][port_name]["SERVER"]["soc_ipv4"] = soc_ipv4_value
 
+def get_tunnel_route_per_port(db, port_tunnel_route, per_npu_configdb, per_npu_asic_db, asic_id, port):
```

Comment:
If the intention is to only display the list of Tunnel route, please fetch data from the APP_DB "TUNNEL_ROUTE_TABLE" and display and avoid this complexity of looping through ASIC DB.

Reply:
Will there be any chance of inconsistency between APP_DB and ASIC_DB? Are there any concerns of using ASIC_DB?

Reply:
There should not be any inconsistency. If it is there, its a bug scenario. You don't have to use the translation logic  for ASIC_DB objects if using APP_DB. Moreover, its straightforward. Tunnel routes is what you are looking for and it is part of the TUNNEL_ROUTE_TABLE.

Reply:
Thanks for clarifying. Will update accordingly.